fdbserver/LogSystem.h

```diff
@@ -237,9 +241,10 @@ class LogSet : NonCopyable, public ReferenceCounted<LogSet> {
 		}
 
 		// the following logic supports upgrades from 5.X
+		int size = SERVER_KNOBS->TLOG_NEW_INTERFACE ? logServersNew.size() : logServers.size();
 		if (tag == txsTag)
-			return txsTagOld % logServers.size();
-		return tag.id % logServers.size();
```

Comment:
the new log system doesn't use tags so the new log system won't trigger this path.